pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -39,6 +39,11 @@ import (
 	volumetypes "k8s.io/kubernetes/pkg/volume/util/types"
 )
 
+const (
+	// EmptyUniquePodName is a UniquePodName for empty string.
```

Comment:
The comment does not say what `EmptyUniquePodName` means. When a volume in ASW/DSW can get `EmptyUniquePodName`? Why is it good for? When you use it in `IsVolumeReconstructed `, how can it happen the pod name is empty, when the volume was reconstructed from the pod directory which contains the pod name?

Reply:
No the pod name can't be empty when volume is reconstructed.  The reason for using empty pod name here is - we are checking for volume state during `nodeStage` (i.e when device is mounted/mapped to global path), so it doesn't matter what is the pod name, but I am trying to see if `uniqueVolumeName` is in reconstructed volume names or not.

Reply:
I mean, it should be written somewhere in a comment, because nobody is going to read this PR. And it's used to _ignore_ pod names for deviceMounts. Would a better name be more appropriate?

And frankly, a whole new function `IsDeviceMountReconstructed` would clear all the confusion about this PR.